src/components/eyewitness/eyewitness.tsx

```diff
@@ -30,7 +32,7 @@ const steps =
     "Step 4: Press the scan button. ";
 const sourceLink = "https://www.kali.org/tools/eyewitness/#eyewitness"; // Link to the source code or relevant documentation.
 const tutorial = ""; // Link to the official tutorial/documentation.
-const dependencies = ["python3"]; // Dependencies required by the component.
+const dependencies = ["Eyewitness"]; // Dependencies required by the component.
```

Comment:
This needs to be lowercase to install correct dependencies.
```suggestion
const dependencies = ["eyewitness"]; // Dependencies required by the component.
```